src/plugins/ui_feature/find_and_replace.ts

```diff
@@ -85,9 +90,20 @@ export class FindAndReplacePlugin extends UIPlugin {
         this.clearSearch();
         break;
       case "ACTIVATE_SHEET":
+        this.userInitiatedSheetChange = true;
```

Comment:
won't work as intended either.
e.g.
1- on sheet 1, write ="test"
2- switch to sheet 2
3- open find & replace side panel
4- start typing 't' -> it says it found one match: ok
5- now continue typing 'e' -> the active sheet now switched to Sheet1

the issue with the boolean approach here is that it will work for 1 handling of the command (step 4° but not for the following steps (step 5)